qa/tasks/ceph_manager.py

```diff
@@ -986,6 +986,7 @@ def do_thrash(self):
             # Allow successful completion so gevent doesn't see an exception.
             # The DaemonWatchdog will observe the error and tear down the test.
 
+    @staticmethod
```

Comment:
we need to update the caller site. otherwise these "log_exc" will be unbound method. in other words, should put.
```py
    @OSDThrasher.log_exc
    def do_sighup(self):
      # ...
```

```
TypeError: 'staticmethod' object is not callable
2020-01-13T14:45:58.296 DEBUG:teuthology.run_tasks:Unwinding manager install
2020-01-13T14:45:58.311 ERROR:teuthology.contextutil:Saw exception from nested tasks
Traceback (most recent call last):
  File "/home/teuthworker/src/git.ceph.com_git_teuthology_master/teuthology/contextutil.py", line 34, in nested
    yield vars
  File "/home/teuthworker/src/git.ceph.com_git_teuthology_master/teuthology/task/install/__init__.py", line 599, in task
    yield
  File "/home/teuthworker/src/git.ceph.com_git_teuthology_master/teuthology/run_tasks.py", line 86, in run_tasks
    manager = run_one_task(taskname, ctx=ctx, config=config)
  File "/home/teuthworker/src/git.ceph.com_git_teuthology_master/teuthology/run_tasks.py", line 64, in run_one_task
    task = get_task(taskname)
  File "/home/teuthworker/src/git.ceph.com_git_teuthology_master/teuthology/run_tasks.py", line 28, in get_task
    module = _import('tasks', module_name, task_name, fail_on_import_error=True)
  File "/home/teuthworker/src/git.ceph.com_git_teuthology_master/teuthology/run_tasks.py", line 52, in _import
    0,
  File "/home/teuthworker/src/git.ceph.com_git_teuthology_master/virtualenv/local/lib/python2.7/site-packages/gevent/builtins.py", line 96, in __import__
    result = _import(*args, **kwargs)
  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-kefu-testing-2020-01-13-1112/qa/tasks/ceph.py", line 21, in <module>
    from ceph_manager import CephManager, write_conf
  File "/home/teuthworker/src/git.ceph.com_git_teuthology_master/virtualenv/local/lib/python2.7/site-packages/gevent/builtins.py", line 96, in __import__
    result = _import(*args, **kwargs)
  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-kefu-testing-2020-01-13-1112/qa/tasks/ceph_manager.py", line 118, in <module>
    class OSDThrasher(Thrasher):
  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-kefu-testing-2020-01-13-1112/qa/tasks/ceph_manager.py", line 1004, in OSDThrasher
    @log_exc
TypeError: 'staticmethod' object is not callable
```

see http://pulpito.ceph.com/kchai-2020-01-13_06:08:15-rados-wip-kefu-testing-2020-01-13-1112-distro-basic-smithi/4664265/

Reply:
@tchaikov done. please retest.